Rakefile

```diff
@@ -102,8 +102,8 @@ if saas?
       end
 
       desc 'Run RSpec test/s specified by input file pattern'
-      RSpec::Core::RakeTask.new(:specific, :test_name) do | t, task_args |
-        t.pattern = "#{task_args[:test_name]}"
+      RSpec::Core::RakeTask.new(:specific, :test_name) do | task, task_args |
```

Comment:
ah, fixed the name here - the style issue remains, and you should squash this commit into the one that introduces these changes.